en-US/Grammar.xml

```diff
@@ -416,9 +454,189 @@
 
 		</table>
 
+<!-- Added section about avoiding causative verbs -->
+		 <formalpara id="causative">
+			<title>Causative Verbs</title>
+			 <para>
+				Avoid the construction "have something happen". Rewrite to replace "have" with a verb that describes the actual action.  
+			</para>
+
+		</formalpara>
+		 <table>
+			<title></title>
+			 <tgroup cols="2" colsep="1" rowsep="1">
+				<colspec colname="c1"></colspec>
+				 <colspec colname="c2"></colspec>
+				 <thead>
+					<row>
+						<entry> Example </entry>
+						 <entry> Improvement </entry>
+
+					</row>
+
+				</thead>
+				 <tbody>
+					<row>
+						<entry> The latter connection has no devices assigned. </entry>
+						 <entry> No devices are assigned to the latter connection. </entry>
+
+					</row>
+					<row>
+						<entry> To have the changes take effect … </entry>
+						 <entry> To apply the changes … </entry>
+
+					</row>
+
+				</tbody>
+
+			</tgroup>
+
+		</table>
+
+<!-- 	 <formalpara id="condition-action">
+			<title>Condition and Action</title>
+			 <para>
+				A condition should precede an action. A clause that explains "why", or for what purpose, should come before a statement of what the action is or how it is done.  
+			</para>
+
+		</formalpara>
+		 <table>
+			<title></title>
+			 <tgroup cols="2" colsep="1" rowsep="1">
+				<colspec colname="c1"></colspec>
+				 <colspec colname="c2"></colspec>
+				 <thead>
+					<row>
+						<entry> Example </entry>
+						 <entry> Improvement </entry>
+
+					</row>
+
+				</thead>
+				 <tbody>
+					<row>
+						<entry> Divide the storage devices into smaller chunks to create a partition. </entry>
+						 <entry> To create a partition, divide the storage devices into smaller chunks. </entry>
+
+					</row>
+
+				</tbody>
+
+			</tgroup>
+
+		</table> -->
+
+<!-- Added section about "either, or" -->
+		 <formalpara id="either-or">
+			<title>The Either-Or Construction</title>
+			 <para>
+				Use the "either x ... or y" construction only to refer to a choice between two options, not for three or more options.
+			</para>
+
+		</formalpara>
+		 <table>
+			<title></title>
+			 <tgroup cols="2" colsep="1" rowsep="1">
+				<colspec colname="c1"></colspec>
+				 <colspec colname="c2"></colspec>
+				 <thead>
+					<row>
+						<entry> Example </entry>
+						 <entry> Improvement </entry>
+
+					</row>
+
+				</thead>
+				 <tbody>
+					<row>
+						<entry> Scale up by adding more resources, with either more memory, CPU, disk space, or other resources. </entry>
+						 <entry> Scale up by adding more resources, with more memory, CPU, disk space, or other resources. </entry>
+
```

Comment:
I would have written "...resources, such as memory, CPUs, disk space, and so on." Memory is not countable, and neither is disk space, but CPU is a countable noun.